dot/node.go

```diff
@@ -64,8 +64,14 @@ func InitNode(cfg *Config) error {
 	)
 
 	// create genesis from configuration file
-	gen, err := genesis.NewGenesisFromJSON(cfg.Init.Genesis)
-	if err != nil {
+	var gen *genesis.Genesis
+	var lErr error
+	if cfg.Init.GenesisHR {
+		gen, lErr = genesis.NewGenesisFromJSONHR(cfg.Init.Genesis)
+	} else {
+		gen, lErr = genesis.NewGenesisFromJSON(cfg.Init.Genesis)
+	}
```

Comment:
not sure how I feel about this, you should only be able to use one of `--genesis` or `--genesis_hr`, not both, could you add some check that makes sure only one is set?

Reply:
updated flag names and check for duplicates.